core/lease/errors.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Canonical Ltd.
```

Comment:
These are just relocated to their own file from separate areas of the code base.